locales/translation.json

```diff
@@ -0,0 +1,693 @@
+{
```

Comment:
Why this file ?

Reply:
this is the source file for blip strings.
language files (en, fr, de...) will be generated by crowdin an we should not modify them